### PR TITLE
Update Simplified Chinese translation.

### PR DESCRIPTION
**Pull request by 872517414**

### CODE REVIEW

File: translations/chi/get5.phrases.txt [hunk 1]
```diff
@@ -226,7 +230,7 @@
     }
     "ChangingMapInfoMessage"
     {
-        "chi"            "切换地图至{GREEN}{1}..."
+        "chi"            "切换地图至{GREEN}{1}......"
```
Comment (Technoblazed) [inline translations/chi/get5.phrases.txt]: What's with the three extra periods?

Comment (872517414) [reply translations/chi/get5.phrases.txt]: In Chinese, the ellipsis is six periods.

Comment (Technoblazed) [reply translations/chi/get5.phrases.txt]: Fair enough

**Comment by splewis** [comment]
Thanks!

**Pull request merged**